### DISCUSSION TITLE
Welcome to HEC-FDA Discussions!

### DISCUSSION THREAD

**Discussion by Brennan1994** [HydrologicEngineeringCenter/HEC-FDA#20] [Announcements]
Welcome to FDA Discussions!

**Comment by rnugent3** [comment]
@HenryGeorgist @Brennan1994 - any thoughts on how to maintain our discussions? Do we leave discussions that have been resolved on our page as a matter of public record (including for ourselves) or delete resolved discussions?

**Reply by Brennan1994** [reply]
I say we leave them. I like that we can go back and see why we made decisions we did, and I kinda like that other people can see that too.

**Comment by rnugent3** [comment]
Sounds good to me. Thanks, Brennan.
